### PR TITLE
Adding additional info regarding the --backend-frontname on Magento installation

### PR DESCRIPTION
**Pull request by eduard13**
## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add some additional information if the _--backend-information_ argument isn't used while installing Magento from command line.

Also it is added an info warning to pay attention while installing the Magento from cli.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/install-gde/composer.html
- https://devdocs.magento.com/guides/v2.2/install-gde/composer.html
- https://devdocs.magento.com/guides/v2.1/install-gde/composer.html

- https://devdocs.magento.com/guides/v2.1/install-gde/install/cli/install-cli-install.html
- https://devdocs.magento.com/guides/v2.2/install-gde/install/cli/install-cli-install.html
- https://devdocs.magento.com/guides/v2.3/install-gde/install/cli/install-cli-install.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/install-gde/install/cli/install-cli-install.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Any options that contain spaces or special characters must be enclosed in either
...
+| `--backend-frontname`            | Uniform Resource Identifier ([URI](http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.2){:target="_blank"}) to access the Magento Admin or omit this parameter to let Magento generate a random URI for you by following pattern <code>admin_jkhgdfq</code>. <br><br>We recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit. <br><br>The URI displays at the end of the installation. You can display it later at any time using the [magento info:adminuri](<{{ page.baseurl }}/install-gde/install/cli/install-cli-adminurl.html>) command. <br><br>If you choose to enter a value, we recommend you not use a common word like admin, backend, and so on. The Admin URI can contain alphanumeric values and the underscore character (\`_\`) only.                                                                                                                                                                                                                                                                          | No        |
```
Comment (erikmarr) [inline guides/v2.1/install-gde/install/cli/install-cli-install.md]:
```suggestion
| `--backend-frontname`            | Uniform Resource Identifier ([URI](http://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.2){:target="_blank"}) to access the Magento Admin. You can omit this parameter to let Magento generate a random URI for you with the following pattern <code>admin_jkhgdfq</code>. <br><br>We recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit. <br><br>The URI displays at the end of the installation. You can display it later at any time using the [magento info:adminuri](<{{ page.baseurl }}/install-gde/install/cli/install-cli-adminurl.html>) command. <br><br>If you choose to enter a value, we recommend you not use a common word like admin, backend, and so on. The Admin URI can contain alphanumeric values and the underscore character (\`_\`) only.                                                                                                                                                                                                                                                                          | No        |
```

File: guides/v2.2/install-gde/install/cli/install-cli-install.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ In Magento Commerce version 2.2.8 and later, you can create the Magento admin us
...
+|`--backend-frontname`|Uniform Resource Identifier (URI) to access the Magento Admin or omit this parameter to let Magento generate a random URI for you by following pattern <code>admin_jkhgdfq</code>.<br><br>We recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit.<br><br>The URI displays at the end of the installation. You can display it later at any time using the magento info:adminuri command.<br><br>If you choose to enter a value, we recommend you not use a common word like admin, backend, and so on. The Admin URI can contain alphanumeric values and the underscore character (`_`) only.|No|
```
Comment (erikmarr) [inline guides/v2.2/install-gde/install/cli/install-cli-install.md]: see comment above

**Comment by eduard13** [comment]
Hi @erikmarr, thank you for suggestions, the content has been adjusted.

**Comment by erikmarr** [comment]
running tests

File: guides/v2.1/install-gde/composer.md [hunk 3]
```diff
@@ -103,6 +103,9 @@ bin/magento setup:install \
 --use-rewrites=1
 ```
 
+{: .bs-callout .bs-callout-info }
+Please note that for `--backend-frontname` we recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit.
```
Comment (erikmarr) [inline guides/v2.1/install-gde/composer.md]:
```suggestion
For `--backend-frontname` we recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit.
```

File: guides/v2.3/install-gde/composer.md [hunk 4]
```diff
@@ -100,6 +100,9 @@ bin/magento setup:install \
 --use-rewrites=1
 ```
 
+{: .bs-callout .bs-callout-info }
+Please note that for `--backend-frontname` we recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit.
```
Comment (erikmarr) [inline guides/v2.3/install-gde/composer.md]:
```suggestion
For `--backend-frontname` we recommend a random URI for security purposes. A random URI is harder for hackers or malicious software to exploit.
```

**Comment by eduard13** [comment]
@erikmarr done.

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @eduard13!

**Pull request merged**